### PR TITLE
Add initial config files for global aerosol DA

### LINKED ISSUE #984: Add/Modify config.* files for JEDI-based Aerosol DA
**Description**
To support global aerosol initialization for future GEFS/GFS implementations, we need to add to global-workflow config.* files to support aerosol DA configuration. The `config.resources` file will also need to be modified as will likely `config.base`.

**Requirements**
- Changes to `config.base` to turn on/off aerosol DA
- Changes to `config.resources` for all new tasks
- Addition of new config files for aerosol DA related options

**Acceptance Criteria (Definition of Done)**
Reviewed by global-workflow team and approved/merged

**Dependencies**
None

### PR DESCRIPTION
**Pull request by CoryMartin-NOAA**
**Description**

Fixes #984 

This PR adds in initial configuration files for support of future global aerosol analysis capabilities.
New files:
```
- config.aeroanl
- config.aeroanlinit
- config.aeroanlrun
- config.aeroanlfinal
```
Modified files:
```
- config.resources
- env/HERA.env
- env/ORION.env
```

**Type of change**

- [x] New feature (non-breaking change which adds functionality)

**How Has This Been Tested?**

- [x] Exercised experiment setup and XML generation using default instructions on wiki
  
**Checklist**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published

### CODE REVIEW

**Comment by CoryMartin-NOAA** [comment]
I created an atm only free forecast test experiment just to confirm that `config.resources` and `env/HERA.env` were not broken by these changes. All new files should not do anything beyond sit in `config/` at this time.

File: parm/config/config.resources [hunk 1]
```diff
@@ -156,6 +157,43 @@ elif [ $step = "atmanalpost" ]; then
...
+      layout_y=6
+    fi
+    if [ $CASE = "C384" ]; then
+      layout_x=5
+      layout_y=5
+    fi
+    if [ $CASE = "C192" -o $CASE = "C96" -o $CASE = "C48" ]; then
+      layout_x=3
+      layout_y=3
+    fi
```
Comment (aerorahul) [inline parm/config/config.resources]: minor suggestion
```suggestion
   case $CASE in
     C768)    
        layout_x=6
        layout_y=6
        ;;
      C384)    
        layout_x=5
        layout_y=5
        ;;
     C192 | C96 | C48)    
        layout_x=3
        layout_y=3
        ;;
    esac
```

File: parm/config/config.resources [hunk 2]
```diff
@@ -156,6 +157,43 @@ elif [ $step = "atmanalpost" ]; then
     export nth_atmanalpost=1
     export npe_node_atmanalpost=$(echo "$npe_node_max / $nth_atmanalpost" | bc)
 
+elif [ $step = "aeroanlinit" ]; then
```
Comment (aerorahul) [inline parm/config/config.resources:160]: are we not going to run this as part of `CDUMP = gfs`?
i.e. Is there no need for the `_gfs` counterparts?

**Review by aerorahul — COMMENTED** [review]
Looks good.
A suggestion and a question.

File: parm/config/config.resources (hunk 2 above)
Comment (CoryMartin-NOAA) [reply parm/config/config.resources:160]: I think exact structure is still TBD. My guess is we will *only* run it as part of the early cycle to initialize GEFS. However, at this phase it's just to cycle so I would run GDAS only in the workflow.I suppose it's easy enough to add `_gfs` here for now just to not paint ourselves into a corner.

File: parm/config/config.resources (hunk 1 above)
Comment (CoryMartin-NOAA) [reply parm/config/config.resources]: Good suggestion, will update.

File: parm/config/config.resources (hunk 2 above)
Comment (aerorahul) [reply parm/config/config.resources:160]: Understood. 
They don't need to be added right now if there is no use case, but if someone tries to configure with _gfs, a warning could/should be added.

Comment (CoryMartin-NOAA) [reply parm/config/config.resources:160]: I've updated it to match what is there for the JEDI ATM work. Let me know if there are additional `gfs` specific things that need added.

**Review by aerorahul — APPROVED** [review]
looks good.

**Pull request merged**